
Drop "Configure Command" from phpinfo as it doesn't
provide any useful information.
The available extensions are not related to this command.

diff -up php-7.0.0RC1/ext/standard/info.c.phpinfo php-7.0.0RC1/ext/standard/info.c
--- php-7.0.0RC1/ext/standard/info.c.phpinfo	2015-08-18 23:39:24.000000000 +0200
+++ php-7.0.0RC1/ext/standard/info.c	2015-08-22 07:56:18.344761928 +0200
@@ -870,9 +870,6 @@ PHPAPI void php_print_info(int flag)
 #ifdef ARCHITECTURE
 		php_info_print_table_row(2, "Architecture", ARCHITECTURE);
 #endif
-#ifdef CONFIGURE_COMMAND
-		php_info_print_table_row(2, "Configure Command", CONFIGURE_COMMAND );
-#endif
 
 		if (sapi_module.pretty_name) {
 			php_info_print_table_row(2, "Server API", sapi_module.pretty_name );
diff -up php-7.0.0RC1/ext/standard/tests/general_functions/phpinfo.phpt.phpinfo php-7.0.0RC1/ext/standard/tests/general_functions/phpinfo.phpt
--- php-7.0.0RC1/ext/standard/tests/general_functions/phpinfo.phpt.phpinfo	2015-08-18 23:39:22.000000000 +0200
+++ php-7.0.0RC1/ext/standard/tests/general_functions/phpinfo.phpt	2015-08-22 07:56:18.344761928 +0200
@@ -20,7 +20,6 @@ PHP Version => %s
 
 System => %s
 Build Date => %s%a
-Configure Command => %s
 Server API => Command Line Interface
 Virtual Directory Support => %s
 Configuration File (php.ini) Path => %s
